### DISCUSSION TITLE
Supported formats?

### DISCUSSION THREAD

**Discussion by ddogfoodd** [swesterfeld/audiowmark#54] [Q&A] [answered]
Is there some documentation on which formats are supported?

**Comment by swesterfeld** [comment, marked as answer]
Since audiowmark uses libsndfile for **reading audio files**, as it supports everything libsndfile does, see

- https://libsndfile.github.io/libsndfile/formats.html

So the popular formats are wav, flac and ogg. There is also support for reading mp3s, regardless of whether your libsndfile library supports it or not.

When **writing audio files**, the output format is always wav or rf64 if you use `--output-format rf64`.
